src/pages/JobLogs.tsx

```diff
@@ -1,34 +1,34 @@
-import { useEffect } from "react";
-import { useQuery, useLazyQuery } from "@apollo/client";
+import { useQuery } from "@apollo/client";
 import styled from "@emotion/styled";
-import Card from "@leafygreen-ui/card";
-import { H2, Subtitle, Body } from "@leafygreen-ui/typography";
+import { H2, Subtitle } from "@leafygreen-ui/typography";
 import { Skeleton } from "antd";
 import { useParams } from "react-router-dom";
 import { useJobLogsAnalytics } from "analytics/joblogs/useJobLogsAnalytics";
-import { StyledRouterLink, StyledLink, PageWrapper } from "components/styles";
+import { Button } from "components/Button";
+import { StyledRouterLink, PageWrapper } from "components/styles";
 import { getLobsterTestLogCompleteUrl } from "constants/externalResources";
 import { getTaskRoute } from "constants/routes";
 import { size } from "constants/tokens";
 import { useToastContext } from "context/toast";
 import {
   GetDisplayTaskQuery,
   GetDisplayTaskQueryVariables,
-  GetTestsQuery,
-  GetTestsQueryVariables,
 } from "gql/generated/types";
-import { GET_DISPLAY_TASK, GET_TESTS } from "gql/queries";
+import { GET_DISPLAY_TASK } from "gql/queries";
 import { PageDoesNotExist } from "pages/404";
+import { JobLogsTable } from "pages/jobLogs/JobLogsTable";
 
 export const JobLogs = () => {
   const dispatchToast = useToastContext();
   const { sendEvent } = useJobLogsAnalytics();
+
```

Comment:
We should had add a page title to this page. We have a hook to do this called `usePageTitle()`